ko/documentation/index.md

```diff
@@ -77,85 +87,172 @@ Ruby를 이해하고 싶다면, [docs.ruby-lang.org][docs-rlo-en]의 권위 있
 [Ruby API][40] (영문)
 : Ruby 클래스, 모듈, 메서드를 편하게 찾고 탐색하세요.
 
+[ruby-doc.org][39]
+: 온라인 API 문서입니다.
+
+[DevDocs.io][40]
+: 온라인 API 문서입니다.
+
+[Ruby QuickRef][42]
+: 빠른 Ruby 참조 문서입니다.
+
+[rubyreferences][43]
+: 완전한 언어 레퍼런스와 상세한 언어 변경 로그입니다. 훌륭합니다.
+
+### 스타일 가이드
+
+[rubystyle.guide][44]
+: RuboCop의 Ruby 스타일 가이드
+
+[RuboCop][45]
+: Rubocop의 스타일 가이드를 자동으로 적용합니다.
+
+[Shopify][46]
+: Shopify의 Ruby 스타일 가이드
+
+[GitLab][47]
+: Gitlab의 Ruby 스타일 가이드
```

Comment:
I think this should be fixed to `GitLab`.
```suggestion
: GitLab의 Ruby 스타일 가이드
```

Reply:
fixed 4e05ac9